.github/workflows/add-to-project.yaml

```diff
@@ -0,0 +1,26 @@
+name: Add issue to project
+
+on:
+  workflow_call:
+    inputs:
+      project_id:
+        type: number
+        required: true
```

Comment:
Do we expect to have more projects? Right now we're using a single one, right? Can we set this one as the default choice?